### PR TITLE
[TogglePassword] Add example page for ux.symfony.com

### PR DESCRIPTION
**Pull request by feymo**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | N/A
| License       | MIT

This PR add the new TogglePassword component to ux.symfony.com with a minimalist example of what to expect "out of the box" 😄  (using default theme without any customization)

### Previews
**Homepage card**
image

**TogglePassword page** 
image

### CODE REVIEW

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends 'packageBase.html.twig' %}
+
+{% block component_header %}
+    {% component PackageHeader with {
+        package: 'toggle-password',
+        eyebrowText: 'Switch between password & text'
+    } %}
+        {% block title_header %}
+            From password to text
+            <span class="playfair ps-2">& vice versa</span>
```
Comment (weaverryan) [inline ux.symfony.com/templates/ux_packages/toggle_password.html.twig]: Minor:

```suggestion
            <span class="playfair ps-2">&amp; vice versa</span>
```

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        {% endblock %}
+
+        {% block sub_content %}
+            Upgrade your password field to toggable field between text and password.
```
Comment (weaverryan) [inline ux.symfony.com/templates/ux_packages/toggle_password.html.twig]: WDYT about:

```suggestion
            Upgrade your password field to toggle between text and password.
```

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+{% endblock %}
+
+{% block code_block_left %}
+    <twig:CodeBlock filename="src/Form/TogglePasswordForm.php" />
+{% endblock %}
+
+{% block code_block_right %}
+    <twig:CodeBlock
+            filename="templates/ux_packages/toggle_password.html.twig"
+            targetTwigBlock="demo_content"
```
Comment (weaverryan) [inline ux.symfony.com/templates/ux_packages/toggle_password.html.twig:26]: Too much indentation on these 2 lines

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    />
+{% endblock %}
+
+{% block demo_title %}UX Toggle Password{% endblock %}
+
+{% block demo_content %}
+    {{ form_start(form) }}
+    {{ form_row(form.email) }}
+    {{ form_row(form.password) }}
+    <button type="submit" class="btn btn-primary">Login</button>
```
Comment (weaverryan) [inline ux.symfony.com/templates/ux_packages/toggle_password.html.twig:36]: I usually indent the `form_row` and `button` - as if `form_start()` was actually a `<form>` tag. Subjective though

**Review by weaverryan — COMMENTED** [review]
👏 You even made/prepped the icon for this the (👁️ ) - really nice work once again

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig (hunk 2 above)
Comment (feymo) [reply ux.symfony.com/templates/ux_packages/toggle_password.html.twig]: Much better indeed 👍🏼 thanks !

**Comment by weaverryan** [comment]
Thanks Félix! I'll deploy this when we push out the next version  it's been a pleasure reviewing your work!

File: ux.symfony.com/templates/ux_packages/toggle_password.html.twig (hunk 4 above)
Comment (feymo) [reply ux.symfony.com/templates/ux_packages/toggle_password.html.twig:36]: I had a discussion with @HeahDude about this (he shares your opinion) and i agree with both of you, i like the logic 👍🏼

**Pull request merged**